velodrome/grafana-stack/README.md

```diff
@@ -1,7 +1,29 @@
+Overview
+========
+
 The goal of this directory is to set-up the following monitoring stack:
 - InfluxDB as the Time-series Database
 - Grafana as the front-end/display
-- Nginx is use as a proxy (mostly to fix CORS issue)
+- Prometheus as a different flavor of time-series database
+- Nginx is used as a proxy (mostly to fix CORS issue)
+
+Testing locally
+===============
+
+You can set-up your own local grafana-stack easily. In order to do that, you
+probably don't need to set-up passwords or even the `nginx` proxy.
```

Comment:
nit: It's not much help to say that I probably don't need to do something. Do I or don't I?